aws/templates/id/id_cognito.ftl

```diff
@@ -90,33 +92,37 @@
                 "Default" : 30
             },
             {
-                "Name" : "allowUnauthIds",
+                "Name" : "allowUnauthenticatedIds",
                 "Default" : false
             }
             {
                 "Name" : "passwordPolicy",
                 "Children" : [
                     {
-                       "Name" : "MinimumLength",
-                       "Default" : "8"
+                       "Name" : "minimumLength",
```

Comment:
Not sure why you reversed the case on these. Everywhere else we use Title case... ?